R/R/apache.sedona-package.R

```diff
@@ -0,0 +1,7 @@
+#' @keywords internal
```

Comment:
Is this needed?

Reply:
This file creates an extra page in the documentation with the description of the package for convenience. (Otherwise, users have to go out of the help).
It is also used by tools to automate certain package features (here lifecycle management).

See very last paragraph of https://roxygen2.r-lib.org/articles/releases/roxygen2-6.0.0.html